independent-projects/qute/pom.xml

```diff
@@ -92,12 +100,6 @@
                 <artifactId>smallrye-reactive-streams-operators-1.0</artifactId>
                 <version>${version.smallrye-reactive-streams-operators}</version>
             </dependency>
-            <dependency>
-                <groupId>org.junit.jupiter</groupId>
-                <artifactId>junit-jupiter</artifactId>
-                <version>${version.junit}</version>
-                <scope>test</scope>
```

Comment:
Can you check that every time this thing is used, it has the proper test scope?

Reply:
Good catch. Fixed the scope in the generator subproject